src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -12,105 +12,325 @@ Sentry's Performance features are currently in beta. For more details about acce
     level="warning"
 %}
 
-Enabling tracing data collection augments your existing error data to capture interactions among your software systems. This lets Sentry tell you valuable metrics about your software health like throughput and latency, as well as expose the impact of errors across multiple systems. Tracing makes Sentry a more complete monitoring solution to diagnose and measure your application health.
+Enabling tracing augments your existing error data by capturing interactions among your software systems. With tracing, Sentry tracks metrics about your software performance, like throughput and latency, as well as displays the impact of errors across multiple systems. Tracing makes Sentry a more complete monitoring solution, helping you both diagnose problems and measure your application's overall health more easily. Tracing in Sentry provides insights such as:
 
-## Tracing & Distributed Tracing
+- What occurred for a specific error event or issue
+- The conditions that cause bottlenecks or latency issues in your application
+- The endpoints or operations that consume the most time
 
-Applications (for example, web applications) typically consist of interconnected components (also called services). For instance, let us assume a modern web application could be composed of the following components separated by the network boundaries:
+## What is Tracing?
 
-- Frontend (Single-Page Application)
-- Backend
-- Background Queue
-- Notification Job
+To begin, a note about what tracing is not: Tracing is not profiling. Though the goals of profiling and tracing overlap quite a bit, and though they are both tools which can be used to diagnose problems in your application, they differ both in terms of what they measure and how the data is recorded.
```

Comment:
```suggestion
To begin, a note about what tracing is not: Tracing is not profiling. Though the goals of profiling and tracing overlap quite a bit, and can be used to diagnose problems in your application, they differ both in terms of what they measure and how the data is recorded.
```

I found the double though awkward. Feel free to ignore if you disagree.

Reply:
I think it works to take out the second "though" if we also get rid of the first comma. (Otherwise, the first "though" still reads to me like it's only modifying the first idea, not both ideas.)

Changed it to `Though the goals of profiling and tracing overlap quite a bit and they can both be used to diagnose problems in your application, they differ both in terms of what they measure and how the data is recorded.`

Reply:
And... changed it back, because it just sounded weird. I kept your tightening up of the language, though - I do think it's better.